algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -784,50 +787,30 @@ abstract class CMSTest[K: CMSHasher: FromIntLike] extends WordSpec with Matchers
       val minWidth = data1.distinct.size
 
       forAll(
-        (Gen.choose(minDepth, 709), "depth"),
-        (Gen.choose(minWidth, 10000), "width"),
+        (Gen.choose(minDepth, maxDepth), "depth"),
+        (Gen.choose(minWidth, maxWidth), "width"),
         (Gen.choose(Int.MinValue, Int.MaxValue), "seed")) { (depth: Int, width: Int, seed: Int) =>
-          val cms1 = TopNCMS.monoid[K](depth, width, seed, 5).create(data1)
-          cms1.heavyHitters should be(Set(1, 2, 3, 4, 5).toK[K])
-
-          val cms2 = TopNCMS.monoid[K](depth, width, seed, 4).create(data1)
-          cms2.heavyHitters should be(Set(2, 3, 4, 5).toK[K])
-
-          val cms3 = TopNCMS.monoid[K](depth, width, seed, 3).create(data1)
-          cms3.heavyHitters should be(Set(3, 4, 5).toK[K])
-
-          val cms4 = TopNCMS.monoid[K](depth, width, seed, 2).create(data1)
-          cms4.heavyHitters should be(Set(4, 5).toK[K])
-
-          val cms5 = TopNCMS.monoid[K](depth, width, seed, 1).create(data1)
-          cms5.heavyHitters should be(Set(5).toK[K])
```

Comment:
Yes. They are tested by the replaced line (we only check one for each of
the checks, but that is fine).
On Mon, Aug 29, 2016 at 14:18 Ben Pence <notifications@github.com> wrote:

> In
> algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala
> <https://github.com/twitter/algebird/pull/541#discussion_r76711645>:
>
> >          (Gen.choose(Int.MinValue, Int.MaxValue), "seed")) { (depth: Int, width: Int, seed: Int) =>
> > -          val cms1 = TopNCMS.monoid[K](depth, width, seed, 5).create(data1)
> > -          cms1.heavyHitters should be(Set(1, 2, 3, 4, 5).toK[K])
> > -
> > -          val cms2 = TopNCMS.monoid[K](depth, width, seed, 4).create(data1)
> > -          cms2.heavyHitters should be(Set(2, 3, 4, 5).toK[K])
> > -
> > -          val cms3 = TopNCMS.monoid[K](depth, width, seed, 3).create(data1)
> > -          cms3.heavyHitters should be(Set(3, 4, 5).toK[K])
> > -
> > -          val cms4 = TopNCMS.monoid[K](depth, width, seed, 2).create(data1)
> > -          cms4.heavyHitters should be(Set(4, 5).toK[K])
> > -
> > -          val cms5 = TopNCMS.monoid[K](depth, width, seed, 1).create(data1)
> > -          cms5.heavyHitters should be(Set(5).toK[K])
>
> Are these being removed because of redundancy?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/twitter/algebird/pull/541/files/973cb4f94e8734ff7b8a201b40c645500b95dd5e#r76711645>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEJdmpCnzN5bUr4g4wzIeZEi3wGaVyKks5qk3bOgaJpZM4JbPXi>
> .
>